bundles/org.openhab.binding.miio/README.md

```diff
@@ -1501,9 +1528,9 @@ Switch childlock "Child Lock" (G_humidifier) {channel="miio:basic:humidifier:chi
 note: Autogenerated example. Replace the id (humidifier) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
 
 ```java
```

Comment:
java?

Reply:
This was requested in review some age ao
https://github.com/openhab/openhab-addons/pull/4780#discussion_r253308014

Reply:
leave as is then